src/neuroconv/datainterfaces/behavior/lightningpose/lightningposeconverter.py

```diff
@@ -135,42 +141,21 @@ def add_to_nwbfile(
         """
         original_video_interface = self.data_interface_objects["OriginalVideo"]
 
-        original_video_metadata = next(
-            video_metadata
-            for video_metadata in metadata["Behavior"]["Videos"]
-            if video_metadata["name"] == self.original_video_name
-        )
-        if original_video_metadata is None:
-            raise ValueError(f"Metadata for '{self.original_video_name}' not found in metadata['Behavior']['Videos'].")
-        metadata_copy = deepcopy(metadata)
-        metadata_copy["Behavior"]["Videos"] = [original_video_metadata]
+        # ExternalVideoInterface expects metadata in ExternalVideos dict format
         original_video_interface.add_to_nwbfile(
             nwbfile=nwbfile,
-            metadata=metadata_copy,
-            stub_test=stub_test,
-            external_mode=external_mode,
+            metadata=metadata,
             starting_frames=starting_frames_original_videos,
+            parent_container="acquisition",
         )
 
         if "LabeledVideo" in self.data_interface_objects:
             labeled_video_interface = self.data_interface_objects["LabeledVideo"]
-            labeled_video_metadata = next(
```

Comment:
Similarly, here we should accept the old metadata structure and then propagate the info to the new metadata structure and raise a FutureWarning if we detect the old metadata structure is being used.